semgrep/semgrep/cli.py

```diff
@@ -108,6 +108,11 @@ def cli() -> None:
         action="store_true",
         help="Scan all files even those ignored by a projects gitignore(s)",
     )
+    parser.add_argument(
+        "--ignore-unknown-extensions",
```

Comment:
```suggestion
        "--skip-unknown-extensions",
```

Ignore can be misinterpreted as "ignore that these extensions are unknown and scan anyway"